### PR TITLE
[Sale Widget] Check if sale widget received invalid parameters

### PR DESCRIPTION
**Pull request by jhesgodi**
# Summary
Screenshot 2024-01-19 at 2 18 08 am
Screenshot 2024-01-19 at 2 17 58 am

### CODE REVIEW

**Review by mimi-imtbl — DISMISSED** [review]

File: packages/checkout/widgets-lib/src/locales/en.json [hunk 1]
```diff
@@ -328,22 +328,26 @@
         "WALLET_REJECTED": {
           "description": "You'll need to approve the transaction in your wallet to proceed.",
           "primaryAction": "Try again",
-          "secondaryAction": "Cancel"
```
Comment (andrearampin) [inline packages/checkout/widgets-lib/src/locales/en.json:331]: shouldn't this be updated in the other langs too?

File: packages/checkout/widgets-lib/src/resources/text/textConfig.ts [hunk 2]
```diff
@@ -364,6 +364,10 @@ export const text = {
           primaryAction: 'Try again',
           secondaryAction: 'Dismiss',
         },
+        [SaleErrorTypes.INVALID_PARAMETERS]: {
+          description: 'Sorry, we\'re unable to process your order. Missing information from client',
+          secondaryAction: 'Dismiss',
+        },
       },
```
Comment (andrearampin) [inline packages/checkout/widgets-lib/src/resources/text/textConfig.ts:371]: @dreamoftrees do we still need to update this file?

File: packages/checkout/widgets-lib/src/locales/en.json (hunk 1 above)
Comment (jhesgodi) [reply packages/checkout/widgets-lib/src/locales/en.json:331]: thanks, this is unwanted rolling back

File: packages/checkout/widgets-lib/src/resources/text/textConfig.ts (hunk 2 above)
Comment (dreamoftrees) [reply packages/checkout/widgets-lib/src/resources/text/textConfig.ts:371]: Good catch @andrearampin @jhesgodi the textConfig should be depreciated and I should remove in a fresh PR. Any text changes should be included in the `locales/en.json` and supported by the i18next hooks like `t()`

Comment (jhesgodi) [reply packages/checkout/widgets-lib/src/resources/text/textConfig.ts]: thank you both. fixed

**Comment by jhesgodi** [comment]
@andrearampin @dreamoftrees @mimi-imtbl Please review again

**Review by mimi-imtbl — APPROVED** [review]

**Review by dreamoftrees — APPROVED** [review]
👍 looks good!

**Pull request merged**